packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeForm.tsx

```diff
@@ -236,6 +236,8 @@ export function BridgeForm(props: BridgeFormProps) {
       gasMultiplier: 1.1,
       sourceChainId: from?.network.toString() ?? '',
       destinationChainId: to?.network.toString() ?? '',
+      token: 'NATIVE',
```

Comment:
Similar to previous comment about `NATIVE` constant - we also have this constant on the widgets-lib side. Note this is also a lowercased string so will need the `toUpperCase()` if uppercased required

Reply:
Thanks, PR updated.